src/components/timeline/OverflowEdgeIndicator.js

```diff
@@ -25,7 +25,7 @@ type State = {
   overflowsOnLeft: boolean,
 };
 
-class OverflowEdgeIndicator extends React.PureComponent<Props, State> {
+class OverflowEdgeIndicatorImpl extends React.PureComponent<Props, State> {
```

Comment:
For that one, you don't need to rename it. You can just do `export class OverflowEdgeIndicator` and remove the last line.